.github/workflows/main.yml

```diff
@@ -13,29 +13,39 @@ on:
   # Allows you to run this workflow manually from the Actions tab
   workflow_dispatch:
   
-before_install:
-  
-  
 # A workflow run is made up of one or more jobs that can run sequentially or in parallel
 jobs:
   test:
     # The type of runner that the job will run on
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-#        os: [ubuntu-18.04, macos-10.15, windows-2019]
+#        os: [ubuntu-18.04, macos-10.15]
```

Comment:
I guess there is nobody doing fortran on windows, but macos might be nice to have? (and if so, also adding instructions to the readme for compiling on macos)

Reply:
I will add macos after this merge in a new branch. I tried to keep the CI run time down but I guess people can comment it out if they don't want it. Compiling pFUnit takes some time unfortunately.